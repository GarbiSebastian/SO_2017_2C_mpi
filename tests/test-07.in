member una
addAndInc una
member prueba
addAndInc prueba
member con
addAndInc con
member una
addAndInc una
member palabra
addAndInc palabra
member repetida
addAndInc repetida
member repetida
member con
member una
member palabra
member prueba
q
