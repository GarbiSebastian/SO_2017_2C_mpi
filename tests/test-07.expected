Comandos disponibles:
  load <arch_1> <arch_2> ... <arch_n>
  addAndInc <string>
  member <string>
  maximum
  q|quit
> El string <una> no está
> Agregado: una
> El string <prueba> no está
> Agregado: prueba
> El string <con> no está
> Agregado: con
> El string <una> está
> Agregado: una
> El string <palabra> no está
> Agregado: palabra
> El string <repetida> no está
> Agregado: repetida
> El string <repetida> está
> El string <con> está
> El string <una> está
> El string <palabra> está
> El string <prueba> está
> Finalizamos ejecución
